packages/amplication-client/src/Workspaces/hooks/useUpgradeButtonData.ts

```diff
@@ -80,6 +74,21 @@ export const useUpgradeButtonData = (
           showUpgradeDefaultButton: false,
           isCompleted: true,
         });
+      } else if (
+        subscription.plan.id === BillingPlan.Enterprise &&
+        !subscription.trialEndDate
+      ) {
+        setUpgradeButtonData({
+          showUpgradeTrialButton: false,
+          showUpgradeDefaultButton: false,
+          isCompleted: true,
+        });
+      } else {
+        setUpgradeButtonData({
+          showUpgradeTrialButton: false,
+          showUpgradeDefaultButton: true,
+          isCompleted: true,
```

Comment:
Maybe we can add `isCompleted: true` to the initial state and remove the else block

Reply:
But we need it to be `isCompleted` false otherwise the initial render won't have the data. am I missing something?